Cargo.toml

```diff
@@ -48,6 +48,8 @@ regex = "1"
 base64 = "0.13"
 sha-1 = "0.9"
 either = "1"
+openat-ext = "0.1.10"
```

Comment:
For openat-ext please drop the patch release number, here, but make it 0.1.6 in the Cargo.lock (so as not to get ahead of Fedora, or @cgwalters can bump the Fedora release if necessary).